### ISSUE TITLE
Scanner doesn't load config, if file has more than 150 lines

### ISSUE THREAD

**Issue by jeevatkm** [brettlangdon/forge#36]
@brettlangdon I ran into strange issue. If configuration file has more than 150 lines, it gets hang doesn't finish loading.

It is happening in File load mode:
```go
cfg, err := forge.ParseFile("example.cfg")
```
However if you break file into two files (below 150 lines) use it via `include` option, it loads successfully.

Also if you read complete file and supply it as string, it loads successfully.
```go
cfgBytes, _ := ioutil.ReadFile("/path/to/file")
cfg, err := forge.ParseString(string(cfgBytes))
```

My understanding is scanner is having issue, looking forward to your inputs.

**Comment by brettlangdon** [comment]
hey @jeevatkm sorry that you are experiencing issues.

I just tested with a basic config with 500 lines and it parsed with no problems.

Can you provide an example of how your config is formatted? The one I tested with was just a flat structure.

```
setting_0 = "0";
setting_1 = "1";
setting_2 = "2";
...
setting_499 = "499";
```

```go
package main

import (
	"fmt"

	"github.com/brettlangdon/forge"
)

func main() {
        // Parse config file "test.cfg"
	settings, err := forge.ParseFile("test.cfg")
	if err != nil {
		panic(err)
	}

	// Convert settings to JSON
	jsonBytes, err := settings.ToJSON()
	fmt.Printf("%s\r\n", string(jsonBytes))
}

```

**Comment by jeevatkm** [comment]
@brettlangdon Thank you for response. 

Sure, please use config files from here. It is not a flat structure.
https://github.com/go-aah/website/tree/master/config

Start with reading `aah.conf` which includes `dev.conf` and `prod.conf`. With your above code snippet. Reading gets hang.

Next is `routes.conf` with your above code snippet, it will throw error or hang.
```
syntax error line <146> column <17>: expected end of section, instead found EOF
```

However, if we read bytes and supply, it works! but `include` tag will be ignored (we know that).
```go
cfgBytes, _ := ioutil.ReadFile("aah.conf")
// cfgBytes, _ := ioutil.ReadFile("routes.conf")

settings, err := forge.ParseBytes(cfgBytes)
if err != nil {
	panic(err)
}
```

**Comment by brettlangdon** [comment]
Thanks for the examples, I'll give them a look over!

**Comment by jeevatkm** [comment]
Okay, I will look forward to it.

**Comment by jeevatkm** [comment]
@brettlangdon any findings? I'm looking forward to it.

**Comment by brettlangdon** [comment]
hey @jeevatkm sorry for the delay, I didn't have a chance to look at this until today. This problem should be resolved on `master` now (new tag `v0.2.3`).

**Comment by jeevatkm** [comment]
@brettlangdon I understand, thank you for your time.
